### DISCUSSION TITLE
Operating Status & Outdoor Air Quality Sensors

### DISCUSSION THREAD

**Discussion by scheelj** [dahlb/ha_carrier#97] [Ideas]
Hello....wondering if you would consider adding the following sensors and entity control. My Carrier Infinity system has these capabilities and it would be wonderful if this integration could be extended to access these capabilities:
- Outdoor Air Quality  Sensor
- Operating Status for AC, Furnace, Fan  (the AC on a multi speed system go from 1-5 and the Furnace is in %)

Thank You
![image_123650291 (2)](https://github.com/dahlb/ha_carrier/assets/75955991/37e27487-3f65-4a3c-a200-0f2d97913c49)
![image_123650291](https://github.com/dahlb/ha_carrier/assets/75955991/a14a3573-504b-463c-bda5-d87d0ea1f155)

**Comment by dahlb** [comment]
![Screenshot_20240228-152118.png](https://github.com/dahlb/ha_carrier/assets/54117/a0ad71d4-9e13-40a5-9080-fe6529bf8aac)

Indoor and outdoor status is already an attribute on the climate entity.

outdoor air quality I have never seen in a carrier API, but if you turn on debug logging as described in the readme and see it let us know and we can expose it through home assistant

**Comment by Aqua1ung** [comment]
Is `hvac_action`  the parameter that tells whether the HVAC is _currently_ idle/cooling/heating? If so, how can this parameter be accessed from the HA interface? I need to design some actions that are constrained by whether the HVAC is currently _idle_ (as opposed to heating/cooling), and I do not know how to incorporate this parameter into the automation.

**Reply by Aqua1ung** [reply]
Nevermind, I figured it out. This is where it's hiding:
![Screenshot From 2024-11-29 20-53-27](https://github.com/user-attachments/assets/1c6b87d8-f7f1-4004-ae18-50593cf4ec61)
